src/js/lib/create-classname-string.js

```diff
@@ -0,0 +1,11 @@
+const createClassNameString = (...classes) =>
```

Comment:
This is fine in my book, only the fileName should be changed 😬  (https://github.com/airbnb/javascript#naming--filename-matches-export)